tests/integration_test_net.go

```diff
@@ -169,6 +171,8 @@ func StartIntegrationTestNetWithJsonGenesis(
 		effectiveOptions.FeatureSet,
 	)
 
+	jsonGenesis.Accounts = append(jsonGenesis.Accounts, effectiveOptions.Accounts...)
```

Comment:
In the function `StartIntegrationTestNetWithFakeGenesis` it should be tested that no extra accounts are specified. If so, the function should panic with a message telling the user that this feature is only supported for the JsonGenesis.